src/main.cpp

```diff
@@ -0,0 +1,75 @@
+#include "illixr.hpp"
+
+#include <csignal>
+
```

Comment:
I think you have to include `iostream` to use `cout`. At least that's the issue in my local machine

Reply:
Thanks for catching this.